src/System.Private.ServiceModel/tools/scripts/sync-pr.cmd

```diff
@@ -60,15 +66,17 @@ if "%3"=="" (
         )
     )
 ) else (
-    set __BRANCH_OR_PR_ID=%3
+    set __BRANCH_OR_PR_ID=%4
     echo   [%~n0] WARNING: This script should usually only be called only from the context of a GitHub Pull Request
-    echo   [%~n0] Branch or PR ID overridden: '%3'
+    echo   [%~n0] Branch or PR ID overridden: '%4'
 ) 
 
-set __REQUEST_URL=%__SYNC_HOST_URL%?%__OPERATION_MODE%=%__BRANCH_OR_PR_ID%
+REM Disregard the chevrons and quotataions in the next line - it's to output the correct ampersand character to PowerShell
+REM We have to dupliate the request URL below for output and for PowerShell because of Powershell's crazy escape sequences interacting 
+REM badly with cmd's crazy and conflicting escape sequences
 
-echo   [%~n0] Making call to '%__REQUEST_URL%'
-powershell -NoProfile -ExecutionPolicy unrestricted -Command "(New-Object Net.WebClient).DownloadString('%__REQUEST_URL%'); 
+echo   [%~n0] Making call to %__SYNC_HOST_URL%?repo=%__REPO_ID%^&%__OPERATION_MODE%=%__BRANCH_OR_PR_ID%
+powershell -NoProfile -ExecutionPolicy unrestricted -Command "(New-Object Net.WebClient).DownloadString('%__SYNC_HOST_URL%?repo=%__REPO_ID%&%__OPERATION_MODE%=%__BRANCH_OR_PR_ID%');"
```

Comment:
"this" refers to the REQUEST_URL. The escaping sequences when combining cmd and PS are nightmarish when you need to escape the & symbol...